journal/updated/07-chapter-6.adoc

```diff
@@ -483,13 +487,13 @@ I should be glad if I could say anything that would be of service to you,
 but you well know that the best help is in yourself.
 O look there--ask of Him who '`gives liberally and upbraids not.`' Neglect
 not your chiefest duty of laying up for yourself treasure in heaven,
-and there is no doubt with me,
+and there is no doubt with me
 that you will then have a comfortable share of earthly blessings.
 
-It is worth while now and then,
+It is worth while now and then
 to look a few years back at the situation your seemingly undone brother
-was in--in whom there was little hope of such a change as we now see nor
-would he ever have so far conquered without a good and strong resolution.
+was in, in whom there was little hope of such a change as we now see;
+he would never have gotten so far without a good and strong resolution.
```

Comment:
I think we need to leave the word "conquer" here, especially because she talks about overcoming the wicked one in the next lines. Maybe: **he never would have so far conquered without a good and strong resolution.**